samcli/commands/build/build_context.py

```diff
@@ -305,8 +302,8 @@ def run(self):
                 )
 
                 click.secho(msg, fg="yellow")
-
         except (
+            FunctionNotFound,
```

Comment:
Should `FunctionNotFound` be in its own `except`? 
because previously it didn't include `click.secho("\nBuild Failed", fg="red")`

Reply:
🤔 makes sense. I was thinking to merge everything, but `FunctionNotFound` is not build failure, since we can't run any build at all. Will update it